packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -222,28 +235,31 @@ const _generateStyles = (light: boolean, placeholder: boolean) => {
     if (light) {
         newStyles = {
             default: {
-                background: "transparent",
+                background: invalid ? tokens.color.fadedRed8 : "transparent",
                 color: placeholder ? white50 : white,
-                borderColor: white50,
+                borderColor: invalid ? Color.red : white50,
                 borderWidth: 1,
             },
             focus: {
-                borderColor: white,
+                borderColor: invalid ? tokens.color.fadedRed8 : white,
                 borderWidth: 2,
                 paddingLeft: adjustedPaddingLeft,
                 paddingRight: adjustedPaddingRight,
             },
             active: {
                 paddingLeft: adjustedPaddingLeft,
                 paddingRight: adjustedPaddingRight,
-                borderColor: mix(fade(blue, 0.32), white),
+                borderColor: invalid ? Color.red : mix(fade(blue, 0.32), white),
```

Comment:
**suggestion:** Now that you are going to import theming, you could use `fadedBlue` here:

```suggestion
                borderColor: invalid ? Color.red : tokens.color.fadedBlue,
```